internal/app/machined/pkg/controllers/kubespan/nftables.go

```diff
@@ -324,6 +363,130 @@ func matchIPSet(set *nftables.Set, mark, mask uint32, family nftables.TableFamil
 	}
 }
 
+func matchIPSetMSS(set *nftables.Set, mtu uint32, family nftables.TableFamily) []expr.Any {
+	var (
+		offset uint32 = 16
+		length uint32 = 4
+		ifmtu  uint16
+		mss    uint16
+	)
+
+	switch {
+	case mtu == 0:
```

Comment:
not sure why `0` can be reached here? 

and default MTU should be 1420.

Reply:
mtu=0 is default value, maybe it was the reason of that.